README.md

```diff
@@ -14,3 +14,18 @@ macOS...sorry :-)
 `hugo new blog/another-blog-post.md`
 
 `hugo new doc/add-this-doc.md`
+
+### Making changes to CSS
+
+The css/sass style code is located in the `themes/kf/sass` directory. If you need
+to make changes to the style, you can update the themes style using:
+
+```
+cd themes/kf/sass
+# Note the npm install step is needed only the first time.
+npm install
+gulp
```

Comment:
npm install doesn't seem to install gulp.

I had to install it separately using `npm install --global gulp-cli`

Can you document this?

Reply:
done